app/views/miq_policy/_event_list.html.haml

```diff
@@ -1,9 +1,11 @@
 #event_list_div
   - if @events
-    = render :partial => "layouts/flash_msg"
```

Comment:
@AparnaKarve That's where it was failing, no message set. We're using info_msg partial elsewhere, better for consistency in code?